gluecodium/src/main/java/com/here/gluecodium/generator/java/JavaModelBuilder.kt

```diff
@@ -335,23 +336,27 @@ class JavaModelBuilder(
         javaInterface.annotations += functionalInterfaceAnnotation
 
         val parameters = limeLambda.parameters.mapIndexed { index, parameter ->
-            JavaParameter(
+            val result = JavaParameter(
                 nameRules.getName(parameter, index),
                 typeMapper.applyNullability(
                     typeMapper.mapType(parameter.typeRef),
                     parameter.typeRef.isNullable
                 )
             )
+            result.comment = createComments(parameter)
+            result
```

Comment:
Is it leftover or funny syntax?

Reply:
It is "funny syntax" to return a value from a block.